templates/common/_base/files/kubelet-auto-sizing.yaml

```diff
@@ -37,7 +37,7 @@ contents:
             recommended_systemreserved_memory=$(echo $recommended_systemreserved_memory 6.72 | awk '{print $1 + $2}')
             total_memory=$((total_memory-112))
         fi
-        if (($total_memory >= 128)); then # 2% of any memory above 128GB
+        if (($total_memory >= 0)); then # 2% of any memory above 128GB
```

Comment:
this was a bug, if the memory was above 128 without this fix it will yield incorrect result.

Reply:
```
memory    reserved 
128           9.32Gi
129           9.34Gi
130           9.36Gi
```
after 128, add 2% of the remaining memory to the reserved.